src/main/java/com/selenium/course/pages/NewAccountForm.java

```diff
@@ -0,0 +1,78 @@
+package com.selenium.course.pages;
+
+import java.util.concurrent.TimeUnit;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebDriverException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.CacheLookup;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import com.selenium.course.framework.DriverManager;
+
+public class NewAccountForm extends CommonForm{
+	private WebDriver driver;
+	private WebDriverWait wait;
+	
+	
+	@FindBy(xpath = "//h2[contains(.,' New Account')]")
+    @CacheLookup
+	WebElement newAccountTitle;
+	
+	@FindBy(name = "save")
+    @CacheLookup
+	WebElement saveBtn;
+	
+	
+	
+	public NewAccountForm() {
+		this.driver = driver;
+		wait = DriverManager.getInstance().getWait();
+		driver = DriverManager.getInstance().getDriver();
+		PageFactory.initElements(driver, this);
+		try {
+			wait.withTimeout(3, TimeUnit.SECONDS).until(
+					ExpectedConditions.visibilityOf(newAccountTitle));
+		} catch (WebDriverException e) {
+			throw new WebDriverException(e);
+		} finally {
+			wait.withTimeout(15, TimeUnit.SECONDS);
+		}
+	}
+	
+	public AccountDetail createNewAccount(String accountName, String parentAccount, String accountNumber, String accountSite, 
```

Comment:
I don't like this method because a good method only have 5 parameters as maximum
Apply a Builder pattern for that